src/model/FanComponentModel.cpp

```diff
@@ -0,0 +1,1520 @@
+    fanEff120CPLANormal.setCoefficient1C1(0.072613);
+    fanEff120CPLANormal.setCoefficient2C2(0.833213);
+    fanEff120CPLANormal.setCoefficient3C3(0.0);
+    fanEff120CPLANormal.setCoefficient4C4(0.013911);
+    fanEff120CPLANormal.setMinimumValueofx(-4.0);
+    fanEff120CPLANormal.setMaximumValueofx(5.0);
+    fanEff120CPLANormal.setMinimumCurveOutput(0.1);
+    fanEff120CPLANormal.setMaximumCurveOutput(1.0);
+    // fanEff120CPLANormal.setInputUnitTypeforx("");
+    // fanEff120CPLANormal.setOutputUnitType("");
+    ok = setNormalizedFanStaticEfficiencyCurveNonStallRegion(fanEff120CPLANormal);
+    OS_ASSERT(ok);
+
+    CurveExponentialSkewNormal fanEff120CPLAStall(model);
+    fanEff120CPLAStall.setName("FanEff120CPLAStall");
+    fanEff120CPLAStall.setCoefficient1C1(-1.674931);
+    fanEff120CPLAStall.setCoefficient2C2(1.980182);
+    fanEff120CPLAStall.setCoefficient3C3(0.0);
+    fanEff120CPLAStall.setCoefficient4C4(1.84495);
+    fanEff120CPLAStall.setMinimumValueofx(-4.0);
+    fanEff120CPLAStall.setMaximumValueofx(5.0);
+    fanEff120CPLAStall.setMinimumCurveOutput(0.1);
+    fanEff120CPLAStall.setMaximumCurveOutput(1.0);
+    // fanEff120CPLAStall.setInputUnitTypeforx("");
+    // fanEff120CPLAStall.setOutputUnitType("");
+    ok = setNormalizedFanStaticEfficiencyCurveStallRegion(fanEff120CPLAStall);
+    OS_ASSERT(ok);
+
+    CurveSigmoid fanDimFlowNormal(model);
+    fanDimFlowNormal.setName("FanDimFlowNormal");
+    fanDimFlowNormal.setCoefficient1C1(0.0);
+    fanDimFlowNormal.setCoefficient2C2(1.001423);
+    fanDimFlowNormal.setCoefficient3C3(0.123935);
+    fanDimFlowNormal.setCoefficient4C4(-0.476026);
+    fanDimFlowNormal.setCoefficient5C5(1.0);
+    fanDimFlowNormal.setMinimumValueofx(-4.0);
+    fanDimFlowNormal.setMaximumValueofx(5.0);
+    fanDimFlowNormal.setMinimumCurveOutput(0.05);
+    fanDimFlowNormal.setMaximumCurveOutput(1.0);
+    // fanDimFlowNormal.setInputUnitTypeforx("");
+    // fanDimFlowNormal.setOutputUnitType("");
+    ok = setNormalizedDimensionlessAirflowCurveNonStallRegion(fanDimFlowNormal);
+    OS_ASSERT(ok);
+
+    CurveSigmoid fanDimFlowStall(model);
+    fanDimFlowStall.setName("FanDimFlowStall");
+    fanDimFlowStall.setCoefficient1C1(0.0);
+    fanDimFlowStall.setCoefficient2C2(5.924993);
+    fanDimFlowStall.setCoefficient3C3(-1.91636);
+    fanDimFlowStall.setCoefficient4C4(-0.851779);
+    fanDimFlowStall.setCoefficient5C5(1.0);
+    fanDimFlowStall.setMinimumValueofx(-4.0);
+    fanDimFlowStall.setMaximumValueofx(5.0);
+    fanDimFlowStall.setMinimumCurveOutput(0.05);
+    fanDimFlowStall.setMaximumCurveOutput(1.0);
+    // fanDimFlowStall.setInputUnitTypeforx("");
+    // fanDimFlowStall.setOutputUnitType("");
+    ok = setNormalizedDimensionlessAirflowCurveStallRegion(fanDimFlowStall);
+    OS_ASSERT(ok);
+
+    // Optional curves
+    // TODO: should these be initialized?
+    ok = assignDefaultOptionalCurves();
+    OS_ASSERT(ok);
```

Comment:
* [ ] Default Ctor initializes Optional Curves from E+ example file... should It? @kbenne

Reply:
Assuming they are not required for simulation, then my two cents is no.